pkg/ddl/reorg.go

```diff
@@ -148,17 +276,6 @@ func reorgTimeZoneWithTzLoc(tzLoc *model.TimeZoneLocation) (*time.Location, erro
 	return tzLoc.GetLocation()
 }
 
-func newReorgSessCtx(store kv.Storage) sessionctx.Context {
```

Comment:
Remove `newReorgSessCtx` from production code because it only used in test now. So we moved it to test and renamed it to `newMockReorgSessCtx `